arg_parser.py

```diff
@@ -0,0 +1,25 @@
+import argparse
+
+class StoreDictKeyPair(argparse.Action):
+	def __call__(self, parser, namespace, values, option_string=None):
+		setattr(namespace, 'models', {kv.split("=", 1)[0]: kv.split("=", 1)[1] for kv in values})
+	
+class ArgParser(object):
+	@staticmethod
+	def parse_args():
```

Comment:
We may want another argument, --label, for classification synset file.

Reply:
Sure, this pr is focusing on open api integration, I will open another PR for that. Maybe you can create an issues to track that.